packages/react/src/progress-indicator/ProgressIndicator.tsx

```diff
@@ -76,8 +76,16 @@ export const ProgressIndicator = ({
 				const isActiveFromLegacyDoingStatus = item.status === 'doing';
 
 				const activePathMatcher = 'href' in item ? item.href : item.label;
+				const activePathMatcherWithTrailingSlash = `${activePathMatcher}${
+					activePathMatcher?.endsWith('/') ? '' : '/'
+				}`;
+				const hasActiveSubStep = !!activePath?.split(
+					activePathMatcherWithTrailingSlash
+				)[1]?.length;
+
 				const isActivePath =
```

Comment:
Just wondering, can all of this be simplified to just this:

```
const isActivePath =
					activePathMatcher === activePath ||
					activePath?.startsWith(`${activePathMatcher}/`);
```

Reply:
There may already be a trailing slash, so it's checking for that and conditionally adding it.